crates/chia-bls/src/bls_cache.rs

```diff
@@ -53,7 +53,9 @@ impl BlsCache {
             // Hash pubkey + message
             let mut hasher = Sha256::new();
             hasher.update(pk.borrow().to_bytes());
-            hasher.update(msg.as_ref());
+            let mut aug_msg = pk.borrow().to_bytes().to_vec();
+            aug_msg.extend_from_slice(msg.as_ref());
+            hasher.update(&aug_msg);
```

Comment:
this is prepending the public key twice. I don't think we need to do that. actually, maybe I was a bit confused when I wrote up that ticket